mmv1/products/compute/Network.yaml

```
@@ -224,9 +224,12 @@ properties:
     default_from_api: true
   - !ruby/object:Api::Type::Enum
     name: 'networkFirewallPolicyEnforcementOrder'
+    update_verb: :PATCH
+    update_url: projects/{{project}}/global/networks/{{name}}
     description: |
-      Set the order that Firewall Rules and Firewall Policies are evaluated. Needs to be either 'AFTER_CLASSIC_FIREWALL' or 'BEFORE_CLASSIC_FIREWALL' Default 'AFTER_CLASSIC_FIREWALL'
-    immutable: true
+      Set the order that Firewall Rules and Firewall Policies are evaluated.
+      Needs to be either `AFTER_CLASSIC_FIREWALL` or `BEFORE_CLASSIC_FIREWALL`.
+      Defaults to `AFTER_CLASSIC_FIREWALL`.
```

```suggestion
      Set the order that Firewall Rules and Firewall Policies are evaluated.
```
Default value and Possible values will be automatically generated in web doc based on the field schema. Web page: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_network#network_firewall_policy_enforcement_order
Therefore let's remove them in this PR to avoid duplicate field info. Thanks!